### PR TITLE
Content change affecting opposite sex marriages in Georgia

### PR DESCRIPTION
**Pull request by ikennaokpala**
[Trello card](https://trello.com/c/ZGJzJx6M/154-georgia-marriage-tool-opposite-sex-marriage-outcomes-content-change-request)

* Content request changes affecting all opposite sex outcomes in Georgia

## Factcheck
[Preview link](https://smart-answers-pr-2568.herokuapp.com/marriage-abroad/y/georgia/third_country/partner_other/opposite_sex)
[GOV.UK](http://gov.uk/marriage-abroad/y/georgia/third_country/partner_other/opposite_sex)

### Before

screen shot 2016-06-02 at 18 17 47

### After

screen shot 2016-06-02 at 18 17 39

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_georgia.govspeak.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  <% if calculator.ceremony_country == 'georgia' && calculator.resident_of_ceremony_country? %>
+
+    ^You should [get legal advice](/government/collections/list-of-lawyers) before making any plans.
+
+  <% else %>
+
+    ^You should [get legal advice](/government/collections/list-of-lawyers) and check the [travel advice for Georgia](/foreign-travel-advice/georgia) before making any plans.^
+
+  <% end %>
```
Comment (chrisroos) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_georgia.govspeak.erb]: Can you change this `if` block to:

```erb
<% if calculator.resident_of_ceremony_country? %>
  <%= render partial: 'get_legal_advice.govspeak.erb' %>
<% else %>
  <%= render partial: 'get_legal_and_travel_advice.govspeak.erb',
             locals: { calculator: calculator } %>
<% end %>
```

We should be trying to reuse existing partials as much as possible to help us avoid introducing more duplication. When I'm making this sort of change, I'll search the project for similar text (e.g. search for "[get legal advice]") to see whether the text has already been extracted to a partial.

**Comment by chrisroos** [comment]
Personally, I would've added Georgia to the regression tests _before_ making any change. This would allow you to make a change and then observe the difference in the regression test artefacts. I think of this as similar to changing an untested piece of code. The first thing I would do in that case is write some tests to describe the current behaviour before changing/adding tests to describe the new behaviour. Does that make sense?

I don't expect you to actually make this change, I'm just explaining what I'd do and why I'd do it.

**Comment by chrisroos** [comment]
Aside from making the change to reuse the existing 'get_legal_advice' and 'get_legal_and_travel_advice' partials, this looks good to me!

**Comment by ikennaokpala** [comment]
@chrisroos will effect the changes..

**Comment by chrisroos** [comment]
Hey @ikennaokpala. Did you want me to look at this PR again or are you still making changes?

**Comment by ikennaokpala** [comment]
@chrisroos not at this point in time.. I think it is already out for factcheck.

**Comment by sistephens** [comment]
Content LGTM 👍

**Comment by ikennaokpala** [comment]
@sistephens thanks for the feedback

**Pull request merged**